Python-Input.md

```diff
@@ -0,0 +1,33 @@
+# Python Input() Function
+
+Many a times, in a program we need some input from the user. Taking inputs from the user makes the program feel a bit interactive. In python, to take input from the user we have a function `input()`. Let's see some examples:
+
+1. When we just want to take the input:
+```python
+# This will just give a prompt without any message
+inp = input()
+```
+[Run Code](https://repl.it/CUqX/0)
+
+2. To give a prompt with message:
+```Python
+prompt_with_message = input('<Your prompt message should appear here>')
+# <Your prompt message should appear here> _
+# The '_' in the output is the prompt
+```
+[Run Code](https://repl.it/CUqX/1)
+
+3. When we want to take an integer input:
+```Python
+number = int(input('Please enter a number: '))
+```
+[Run Code](https://repl.it/CUqX/2)
```

Comment:
Maybe you can mention here what error would be raised if one were to enter something other than a number or its string equivalent?